Sources/Util/IndexTree.swift

```diff
@@ -299,7 +299,7 @@ extension IndexTreeNode {
     /**
      * `insertAt` inserts the given node at the given offset.
      */
-    func insertAt(newNode: Self, offset: Int) throws {
+    func insertAt(_ newNode: Self, _ offset: Int) throws {
```

Comment:
If `insertAt inserts the given node at the given offset.` then probably better if we rename it to
```swift
func insert(_ newNode: Self, at offset: Int)
```